src/utils/state.ts

```diff
@@ -57,7 +57,7 @@ export const getStateActions = <State>({
                     ...config.fetch,
                 },
                 {
-                    track: config.nostoAnalytics ? "autocomplete" : undefined,
+                    track: (config.nostoAnalytics ? "autocomplete" : undefined)!,
```

Comment:
This signature looks a bit weird. The condition returns undefined but we are using bang operator on top of it